numpy/core/fromnumeric.py

```diff
@@ -2120,6 +2122,19 @@ def sum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue,
     >>> np.sum([])
     0.0
 
+    The numerical precision of sum (and ``np.add.reduce``) is in general
```

Comment:
Worth noting this statement only applies to floating point values. Integers are safe.